src/Emulator.cpp

```diff
@@ -123,6 +123,11 @@ bool Emulator::Initialize()
 
     // Create CPU
     m_cpu = std::make_unique<CPU>();
+    if (m_cpu == nullptr)
```

Comment:
This is possible if there is a bad alloc or something. I found mixed documentation that implied it might throw a std::badalloc.  I'm not sure. We could make CPU allocate 50GB of memory and see if fail, but it's not that important. Typically if the machine is low on memory, then lots of stuff will fail.